README.md

```diff
@@ -1150,14 +1144,10 @@ Bookmarks
 
 <dl>
 
-[CS50](https://cs50.harvard.edu/2018/fall/)
-
 [CS50's Introduction to Computer Science | edX](https://www.edx.org/course/cs50s-introduction-computer-science-harvardx-cs50x)
 
 [CS50 - Reference](https://reference.cs50.net/)
 
-[Introduction to Game Development](https://cs50.github.io/games/)
```

Comment:
While this link is not active anymore, there are suitable replacements:
https://cs50.harvard.edu/games/2018/
or https://www.edx.org/learn/game-development/harvard-university-cs50-s-introduction-to-game-development